mindmaps-engine/src/main/java/io/mindmaps/engine/postprocessing/BackgroundTasks.java

```diff
@@ -109,7 +103,7 @@ private void performCastingFix() {
 
             MindmapsGraph graph;
             try {
-                graph = GraphFactory.getInstance().getGraph(entry.getKey());
+                graph = MindmapsClient.getGraph(entry.getKey());
```

Comment:
This should remain `GraphFactory.getInstance()` otherwise postprocessing will submit commit logs to itself. Also MindmapsClient hits the rest API to create the graph.